test/cypress/integration/widgets/filters_spec.ts

```diff
@@ -720,4 +723,44 @@ describe('Filters widget', () => {
             });
         });
     });
+
+    it('should handle all filter parameters', () => {
+        enum RuleParameters {
+            RuleAttributes = 'attributes',
+            RuleAttributesOperators = 'attributesOperators',
+            RuleLabelsOperators = 'labelsOperators',
+            RuleTypes = 'types'
+        }
+        type SupportedByObject = Record<RuleParameters, string[]>;
+
+        function getEnumValues(enumType: any) {
+            return Object.keys(enumType).map(enumValue => enumType[enumValue]);
+        }
+
+        cy.cfyRequest('/searches/deployments.help.json').then(response => {
+            const {
+                operations: [operations]
+            } = response.body;
+
+            const supportedByApi: SupportedByObject = {
+                [RuleParameters.RuleAttributes]: operations.allowed_filter_rules_attrs,
+                [RuleParameters.RuleAttributesOperators]: operations.filter_rules_attributes_operators,
+                [RuleParameters.RuleLabelsOperators]: operations.filter_rules_labels_operators,
+                [RuleParameters.RuleTypes]: operations.filter_rules_types
+            };
+
+            const supportedByUi: SupportedByObject = {
+                [RuleParameters.RuleAttributes]: getEnumValues(FilterRuleAttributes),
+                [RuleParameters.RuleAttributesOperators]: getEnumValues(AttributesFilterRuleOperators),
+                [RuleParameters.RuleLabelsOperators]: getEnumValues(LabelsFilterRuleOperators),
+                [RuleParameters.RuleTypes]: getEnumValues(FilterRuleType)
+            };
+
+            getEnumValues(RuleParameters).forEach(parameter => {
```

Comment:
We could simplify it :smile: 

```suggestion
            Object.values(RuleParameters).forEach(parameter => {
```

and then we don't need `as RuleParameters` below :slightly_smiling_face: